templates/channels/channel_configuration.haml

```diff
@@ -639,6 +639,20 @@
     %code
       https://{{ domain }}{% url 'handlers.chikka_handler' object.uuid %}
 
+
+  -elif object.channel_type == 'TW'
+    %h4
+      -blocktrans
+        To finish configuring your connection you'll need to set the following status callback URL in your Adhearsion Application
+
+    %hr
+
+    %h4
+      -trans "Status Callback URL"
+
+    %code
+      https://{{ domain }}{% url 'handlers.twilio_handler' %}
+
```

Comment:
Looks like you potentially create API tokens as well when adding a new channel, seems those should be displayed here.